forge-gui/res/cardsfolder/upcoming/sadistic_shell_game.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sadistic Shell Game
+ManaCost:4 B
+Types:Sorcery
+A:SP$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBDestroy | InTurnOrder$ True | StackDescription$ SpellDescription | SpellDescription$ Starting with the next opponent in turn order, each player chooses a creature you don’t control. Destroy the chosen creatures.
+SVar:DBChooseCard:DB$ ChooseCard | Choices$ Creature | Reveal$ True | Defined$ Player.IsRemembered | ControlledByPlayer$ Player.Opponent | Mandatory$ True | ChoiceTitle$ Choose a creature to destroy | RememberChosen$ True | AILogic$ OppPreferred
```

Comment:
ControlledByPlayer incorrect

Reply:
I'll look at some other examples of `ControlledByPlayer$`

Reply:
Addressed in recent [commit ](https://github.com/Card-Forge/forge/pull/6226/commits/261308bf219e657dff36908625ce7760906fa91a)

Reply:
nope, that filtering is not what the card says
you should only use `Choices$ Creature.YouDontCtrl`

Reply:
Ah! Sorry. That makes much more sense. I will get better at this as I look into more cards and the code itself!